osu.Game/Overlays/NowPlayingOverlay.cs

```diff
@@ -278,12 +294,34 @@ protected override void Update()
                 progressBar.EndTime = track.Length;
                 progressBar.CurrentTime = track.CurrentTime;
 
+                // Process the Current Minute and Second
+                CurrentMinutes = Math.Floor((track.CurrentTime / 1000) / 60).ToString();
```

Comment:
Please read https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings and reconsider whether you want to be doing this math manually.